File: MokuraMQTT/app/src/main/java/com/example/mokuramqtt/database/Hardware.kt
package com.example.mokuramqtt.database

import android.os.Parcelable
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.parcelize.Parcelize

@Entity(tableName = "hardware")
@Parcelize
data class Hardware(
    @PrimaryKey(autoGenerate = true)
    @ColumnInfo(name = "idHardware")
    val idHardware: Int? = null,

    @ColumnInfo(name = "hardwareSerial")
    val hardwareSerial: String,

    @ColumnInfo(name = "hardwareName")
    val hardwareName: String,

) : Parcelable

File: docker/paho/Hardware.py
class Hardware:
    def __init__(self, id_hardware, hardware_name, hardware_serial):
        self.id_hardware = id_hardware
        self.hardware_name = hardware_name
        self.hardware_serial = hardware_serial

    def to_dict(self):
        return {
            "id_hardware": self.id_hardware,
            "hardware_name": self.hardware_name,
            "hardware_serial": self.hardware_serial
        }